core/src/main/java/org/infinispan/xsite/irac/DefaultIracManager.java

```diff
@@ -531,4 +557,103 @@ synchronized boolean isCompleted() {
+   public int getQueueSize() {
+      return updatedKeys.size();
+   }
+
+   @ManagedAttribute(description = "Number of Conflicts", displayName = "Number of Conflicts", measurementType = MeasurementType.TRENDSUP)
+   public long getNoOfConflicts() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return conflictsCounts.longValue();
+   }
+
+   @ManagedAttribute(description = "Number of Discards", displayName = "Number of Discards", measurementType = MeasurementType.TRENDSUP)
+   public long getNoOfDiscards() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return discardCounts.longValue();
+   }
+
+   @ManagedAttribute(description = "Number of merge policy discard update (local value wins)", displayName = "NumberOfConflictLocalWins", measurementType = MeasurementType.TRENDSUP)
+   public long getNumberOfConflictLocalWins() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return numberOfConflictLocalWins.longValue();
+   }
+
+   @ManagedAttribute(description = "Number of merge policy applies update (remote value wins)", displayName = "NumberOfConflictRemoteWins", measurementType = MeasurementType.TRENDSUP)
+   public long getNumberOfConflictRemoteWins() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return numberOfConflictRemoteWins.longValue();
+   }
+
+   @ManagedAttribute(description = "Number of merge policy created a new value (merge remote value with local value)", displayName = "NumberOfConflictMerged", measurementType = MeasurementType.TRENDSUP)
+   public long getNumberOfConflictMerged() {
+      if (!getStatisticsEnabled()) {
+         return -1;
+      }
+      return numberOfConflictMerged.longValue();
+   }
+
+   @ManagedAttribute(description = "Enables or disables the gathering of statistics by this component", writable = true)
+   @Override
+   public boolean getStatisticsEnabled() {
+      return statisticsEnabled;
+   }
+
+   /**
+    * @param enabled whether gathering statistics for JMX are enabled.
+    */
+   @Override
+   public void setStatisticsEnabled(boolean enabled) {
+      statisticsEnabled = enabled;
+   }
+
+   /**
+    * Resets statistics gathered. Is a no-op, and should be overridden if it is to
+    * be meaningful.
+    */
+   @ManagedOperation(displayName = "Reset Statistics", description = "Resets statistics gathered by this component")
+   @Override
+   public void resetStatistics() {
+      discardCounts.reset();
+      conflictsCounts.reset();
+      numberOfConflictLocalWins.reset();
+      numberOfConflictRemoteWins.reset();
+      numberOfConflictMerged.reset();
+   }
+
+   @Override
+   public void incrementDiscards() {
+      discardCounts.increment();
+   }
+
+   @Override
+   public void incrementConflicts() {
+      conflictsCounts.increment();
+   }
+
+   @Override
+   public void increaseNumberOfConflictLocalWins() {
```

Comment:
make the method's name consistent:
increase => increment

note: same for the others "increase*" methods.

Reply:
Push the changes